src/components/table.tsx

```diff
@@ -199,14 +199,16 @@ const TableRow = memo(
   <RowType extends any>({
```

Comment:
There is no longer a benefit for `TableRow` to be `memo(`'d anymore because its props change on pretty much every sort. Let's remove `memo(`.

Reply:
Good point